guides/performance.md

```diff
@@ -100,9 +100,9 @@ Always remember that it's not just the time that has improved: memory usage is a
 
 ### Use string interpolation instead of concatenation
 
-Sometimes you need to work directly with strings built from combining string literals with other values. You shouldn't just concatenate these strings with `String#+(String)` but rather use [string interpolation](syntax_and_semantics/literals/string.html) which allows to embed expressions into a string literal: `"Hello, #{name}"` is better than `"Hello, " +  name.to_s`.
+Sometimes you need to work directly with strings built from combining string literals with other values. You shouldn't just concatenate these strings with `String#+(String)` but rather use [string interpolation](https://crystal-lang.org/docs/syntax_and_semantics/literals/string.html#interpolation) which allows to embed expressions into a string literal: `"Hello, #{name}"` is better than `"Hello, " +  name.to_s`.
```

Comment:
Please don't use absolute links. It should just be `../syntax_and_semantics/literals/string.html`